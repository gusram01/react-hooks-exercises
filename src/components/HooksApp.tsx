import React from 'react';

const HooksApp = () => {
  return (
    <h1>Hello</h1>
  );
};

export default HooksApp;