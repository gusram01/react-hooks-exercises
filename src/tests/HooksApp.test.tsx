import React from 'react';
import { shallow } from 'enzyme';
import HooksApp from '../components/HooksApp';

describe('HooksApp tests', () => {

  test('HooksApp should render correctly', () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });

});
