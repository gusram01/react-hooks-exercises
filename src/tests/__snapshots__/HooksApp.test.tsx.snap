// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HooksApp tests HooksApp should render correctly 1`] = `
<h1>
  Hello
</h1>
`;

exports[`HooksApp tests should HooksApp render correctly 1`] = `
<h1>
  Hello
</h1>
`;
